package.json

```
@@ -15,7 +15,7 @@
   "scripts": {
     "lint": "npm run lint:wc && npm run lint:js",
     "lint:js": "eslint . --ext .js,.html test/**/*.js test/**/*.html demo/**/*.js demo/**/*.html",
-    "lint:wc": "polymer lint -i \"./*\" \"test\" \"demo\"",
+    "lint:lit-analyzer": "lit-analyzer src/**/* test/**/* stories/web-components/**/* --strict --rules.no-unknown-tag-name off --rules.no-unknown-event error --maxWarnings 0",
```

```suggestion
    "lint:lit-analyzer": "lit-analyzer *.js demo/**/* --strict --rules.no-unknown-tag-name off --rules.no-unknown-event error --maxWarnings 0",
```

Plus update any references to `lint:wc`, or just keep the name of this script (wc is just for "web components" so, generic), and install `lit-analyzer` as a dev dependency, and the entire test flow works correctly on a fresh checkout.